Jenkinsfile

```diff
@@ -72,6 +73,18 @@ pipeline {
                     steps {
                         sh('''
 ./tools/build_jenkins -u 5417ab20-3156-11ea-8b19-2b66f5e7a439 -p $MIN_PLATFORM_STAMP
+        ''')
+                    }
+                }
+                stage('minimal-64-lts 21.4.0') {
+                    agent {
+                        node {
+                            label joyCommonLabels(image_ver: '19.4.0')
```

Comment:
Copy-paste error, shouldn't this be 21.4.0?

Reply:
Yep, and that invalidates my tests.